src/MigrationRunner.test.ts

```diff
@@ -1,9 +1,9 @@
 import Docker from "dockerode"
+import massive from "massive"
+import {IDatabase} from "pg-promise"
```

Comment:
is linting not running here?  this is the only import without spaces around the destructured variable

Reply:
` tslint -c tslint.json src/**/*.ts` It seems like it is being run there, so maybe that's not a rule we're enforcing?